subworkflows/nf-scil/preproc_t1/main.nf

```diff
@@ -36,28 +40,46 @@ workflow PREPROC_T1 {
         ch_versions = ch_versions.mix(IMAGE_RESAMPLE.out.versions.first())
 
         // ** Brain extraction ** //
-        ch_bet = IMAGE_RESAMPLE.out.image.join(ch_template).join(ch_probability_map)
-        BETCROP_ANTSBET ( ch_bet )
-        ch_versions = ch_versions.mix(BETCROP_ANTSBET.out.versions.first())
+        if ( val_synth ) {
```

Comment:
```suggestion
        if ( params.run_synth_bet ) {
```

Reply:
I believe it's alright. Changed the parameter name to run_synth since it will be called globally (not just for synthbet). Maybe we could considered naming it different - run_surgeryflow for instances - since other tools not called "synth" might want to be triggered by it (future bids management or easyreg for instances)